README.md

```diff
@@ -1,2 +1,135 @@
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Portfolio-Setup-and-mobile-first] <a name="about-project"></a>
+
+**[Portfolio-Setup-and-mobile-first]** This is the Jose Delphin Portfolio, that allows you understand what are my skills as a developer.
+
+## 🛠 Built With <a name="Visual Studio Code"></a>
+
+### Tech Stack <a name="Viual Studio Code"></a>
+
+### Key Features <a name="key-features"></a>
+
+- HTML
+- CSS
+- Visual Studio Code 
+- Figma
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Clone the repository using git, and open it with visual studio code.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  cd "your-folder"
+  git clone git@github.com:https:/adelphinsucasa/Portfolio-Setup-and-mobile-first.git
+
+
+### Usage
+
+To open the project, use yout prefered explorer and open the index.html file 
+
+
+### Deployment
+
+You can deploy this project using visual studio code and Figma.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Jose Delphin**
+
+- GitHub: [@githubhandle](https://github.com/adelphinsucasa/Hello_Microverse.git)
+- Twitter: [@twitterhandle](@josedelphin)
+- LinkedIn: [LinkedIn](@adelphin)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can contact the author througt the above meddia links or by email to dalejandro.005@gmail.com
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse and all the persons that got in touch with me while we were working on this project
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ <a name="faq"></a>
+
+- Is it difficult?
+
+  - If you have knowledge about github basic commands, installing npm extension in visual studio code, figma, HTML and CSS it is easier, in other way, it can be little complicated
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ Required] On the readme file the link to the project LICENSE is not working, this is because your project does not have a file named LICENSE. Please add this file to the project and make sure its contents are those of the [MIT License](https://choosealicense.com/licenses/mit/) on this [link](https://choosealicense.com/licenses/mit/). Remember to customize the `year` and `full name` fields of the License.

Reply:
Yes. You are right.
Understood.
Thanks so much for your help.